ec2-userdata/ecs-instance.tpl

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+sudo mkfs -t xfs ${docker_storage_volume_device_name}
+sudo mkdir -p /var/lib/docker
+sudo mount -o prjquota ${docker_storage_volume_device_name} /var/lib/docker
```

Comment:
```
sudo mount -o prjquota ${docker_storage_volume_device_name} /var/lib/docker
                       ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.
```

Reply:
This is a terraform variable too